File: Java/Eliminacion_inicial.java
import java.util.ArrayList;
import java.util.Arrays;

public class Eliminacion_inicial {
   public static void main(String[] args) 
    {
        //Declaracion de array que pueda ser cambiado (dinamico)
        ArrayList<Integer> eliminacion = new ArrayList<>(Arrays.asList(11, 21, 31, 41, 51, 61));

        System.out.print("Antes de la eliminacion al final, el array es: ");

        //variable que guarda el largo del array
        //para saber cual es el último
        int largo = eliminacion.size();

        //For para recorrer el array (sin eliminar nada)
        for (int indice = 0; indice < largo; indice++) {
            System.out.print(eliminacion.get(indice) + " ");
        }
        System.out.println();
        
        //Función para eleminar un elemento ubicado al ultimo del array
        eliminacion.remove(largo - 1);
        
        //Recorre el arreglo final, ya con el elemento de menos
        System.out.print("Después de la eliminacion, el array es: ");
        for (int indice = 0; indice < largo; indice++) {
            System.out.print(eliminacion.get(indice) + " ");
        }
        System.out.println();
    } 
}

File: JavaScript/Eliminacion_inicial.js
// Declaracion de array
const eliminacion = [11, 21, 31, 41, 51, 61];

// Variable que va a guardar el largo del array
const largo = eliminacion.length;

//Recorrer el antes
console.log("El arreglo antes de la eliminacion inicial es: ");
for (let indice = 0; indice < largo; indice++) {
    console.log(eliminacion[indice] + " ");
}
// elimina el primer elemento del array
eliminacion.shift();

// recorrido del resultado
console.log("El arreglo antes de la eliminacion inicial es: ");
for (let indice = 0; indice < largo; indice++) {
    console.log(eliminacion[indice] + " ");
}

console.log();